session/bootstrap.go

```diff
@@ -1489,6 +1493,14 @@ func upgradeToVer69(s Session, ver int64) {
 	doReentrantDDL(s, CreateGlobalGrantsTable)
 }
 
+func upgradeToVer70(s Session, ver int64) {
+	if ver >= version70 {
+		return
+	}
+	doReentrantDDL(s, "ALTER TABLE mysql.user ADD COLUMN plugin CHAR(64) AFTER authentication_string", infoschema.ErrColumnExists)
```

Comment:
do we need set `plugin` column's default value be 'mysql_native_password'?

maybe there are a rare situation: it has two tidb instances, 1st tidb upgraded and add new column, but 2nd tidb hadn't upgrade but some user run grant on it, it maybe insert a empty string to plugin column

Reply:
This case is handled by assuming `mysql_native_password` if the plugin column is empty.

This was suggested by @morgo in https://github.com/pingcap/tidb/pull/24894#discussion_r639846266